### DISCUSSION TITLE
Hola  acabo de subir mi proyecto de pruba a github por medio de comandos en bash . pero no puedo verlo en la pestaña de codigos . Alguien me puede ayudar con alguna sugerencia que estoy haciendo mal.

### DISCUSSION THREAD

**Discussion by JORGEDAN** [community/community#27350] [Other Feature Feedback, Questions, & Ideas]
Hola  acabo de subir mi proyecto de pruba a github por medio de comandos en bash . pero no puedo verlo en la pestaña de codigos . Alguien me puede ayudar con alguna sugerencia que estoy haciendo mal.

**Comment by sawyerbivens** [comment]
**english**
By codes tab do you mean "respositories"?

**spanish (google translate)**
¿Por la pestaña de códigos te refieres a "repositorios"?
